analysers/analyser_osmosis_highway_tunnel_bridge.py

```diff
@@ -101,8 +101,57 @@
     0 = SUM(CASE WHEN layer IS NULL THEN 0 ELSE 1 END) + CASE WHEN blayer IS NULL THEN 0 ELSE 1 END
 """
 
+sql40 = """
+SELECT DISTINCT
+    nodes.id,
+    bt_ways.id,
+    bt_connections.id,
+    ST_AsText(nodes.geom),
+    CASE
+        WHEN bt_ways.tags?'bridge' AND bt_ways.tags->'bridge'!='no' THEN 'bridge'
+        ELSE 'tunnel'
+    END
+FROM
+    {0}highways AS bt_ways
+    JOIN {1}highways AS bt_connections ON
+        bt_connections.linestring && bt_ways.linestring AND
+        bt_connections.nodes && bt_ways.nodes AND
+        bt_connections.id != bt_ways.id
+    JOIN nodes ON
+        nodes.geom && bt_connections.linestring AND nodes.geom && bt_ways.linestring AND -- One is redundant, but let the planner choose
+        nodes.id = ANY(bt_ways.nodes) AND
+        nodes.id = ANY(bt_connections.nodes) AND
```

Comment:
To be checked. Byt may use `<@` operator in place.

Reply:
It seems it's not defined for this data type, I get this warning:
`operator does not exist: bigint <@ bigint[]`